os/arch/Kconfig

```diff
@@ -13,6 +13,7 @@ config ARCH_ARM
 	select ARCH_HAVE_VFORK
 	select ARCH_HAVE_STACKCHECK
 	select ARCH_HAVE_CUSTOMOPT
+	select ARCH_HAVE_NESTED_INTERRUPT
```

Comment:
You only modify armv7-m. Is it ok for armv7-r with this?

Reply:
Currently I have implemented for armv7-m only. We will check feasibility for armv7-r later.

Reply:
1. So, it does not work with armv7-r. But you added selection under ARM, not armv7-m. Is it ok to use armv7-r now?
2. Please let me know the ETA of armv7-r.

Reply:
Hi @sunghan-chang, 

1) I checked armv7-r is working fine as "ARCH_NESTED_INTERRUPT" macro is disable by default.
2) I will be able to complete the changes of armv7-r most probable by 5th june